rest/views.py

```diff
@@ -89,8 +90,15 @@ class VocabularyList(ListCreateAPIView):
 
     def get_queryset(self):
         """Filter to vocabularies for a repository"""
```

Comment:
Out of scope question: What does this description mean? "Filter to vocabularies..."?